source/containers.tex

```diff
@@ -76,8 +76,8 @@
 \tcode{T}, \tcode{a} and \tcode{b}
 denote values of type \tcode{X}, \tcode{u}
```

Comment:
`b` is used in `a.swap(b)` so must be non-const. I have no idea why we have `b` and `r` described differently. I'll do some archaeology.